### DISCUSSION TITLE
2 questions

### DISCUSSION THREAD

**Discussion by skyw33** [firstof9/openevse#52] [Q&A] [answered]
First of all, thank you to the developers!  I have 2 questions:
1) Do you intend to try to get this to replace the broken core integration?
2) Can you add HACS integration?

**Comment by firstof9** [comment, marked as answer]
1. I already tried to get a PR in to begin the process, it never bore fruit.
2. You can add it as a custom repo in HACS. As it's a drop in replacement for a core integration, I don't think it'll be allowed to be added to HACS without modification.
